src/main/java/io/cdap/plugin/gcp/common/GCPUtils.java

```diff
@@ -49,11 +64,55 @@ public static ServiceAccountCredentials loadServiceAccountCredentials(String pat
     }
   }
 
+  public static ServiceAccountCredentials loadServiceAccountCredentials(String content,
+                                                                        boolean isServiceAccountFilePath)
+    throws IOException {
+    if (isServiceAccountFilePath) {
+      return loadServiceAccountCredentials(content);
+    }
+    InputStream jsonInputStream = new ByteArrayInputStream(content.getBytes());
+    return ServiceAccountCredentials.fromStream(jsonInputStream);
+  }
+
+  public static String extractPrivateKey(ServiceAccountCredentials credentials) {
+    return String.format(PRIVATE_KEY_WRAP,
```

Comment:
why are we having this private key wrap? Wouldnt it always have this wrap in the service account key? We should not modify anything in private key

Reply:
``ServiceAccountCredentials.getPrivateKey().getEncoded()`` will not give the private key with prefix ``-----BEGIN PRIVATE KEY-----`` xxxxx ``----- END PRIVATE KEY-----``, is there any other way how get the key from ServiceAccountCredentials?

Reply:
I see